plugin/src/software/aws/toolkits/eclipse/amazonq/chat/ChatCommunicationManager.java

```diff
@@ -431,14 +435,46 @@ private CompletableFuture<Void> handleCancellation(final String tabId) {
 
     private void sendErrorToUi(final String tabId, final Throwable exception) {
         String errorTitle = "An error occurred while processing your request.";
-        String errorMessage = String.format("Details: %s", exception.getMessage());
+        String errorMessage = extractErrorMessage(exception);
+        // Convert literal \n characters to proper line breaks for UI display
+        // Language server sends error messages with \n\n patterns (e.g., "Error message \n\nRequest ID: 123")
+        errorMessage = errorMessage.replace("\\n", System.lineSeparator());
```

Comment:
is this needed for all or just the specific error message where we have request ID?